__tests__/app.test.js

```diff
@@ -242,3 +254,89 @@ describe("POST /api/articles/article_id:/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/article_id:", () => {
+  test("200: Serves the article with the incremented votes", () => {
+    return db
+      .query(`SELECT votes FROM articles WHERE article_id = 1`)
+      .then(({ rows }) => {
+        return request(app)
+          .patch("/api/articles/1")
+          .send({
+            inc_votes: 27,
+          })
+          .expect(200)
+          .then(({ body: { article } }) => {
+            expect(article).toMatchObject({
+              article_id: expect.any(Number),
+              title: expect.any(String),
+              topic: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              created_at: expect.any(String),
+              votes: rows[0].votes + 27,
+              article_img_url: expect.any(String),
+            });
+          });
+      });
+  });
```

Comment:
Nice test and a really good way of asserting the dynamic value of votes after the request, but I would create a variable for `votes[0]` or alternatively you could do a get request to the endpoint `/api/articles/1` which also gives you an article object with a votes key